src/DotNetWorker/Hosting/WorkerMiddlewareWorkerApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.Azure.Functions.Worker.OutputBindings;
+using Microsoft.Azure.Functions.Worker.Pipeline;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Hosting
+{
+    /// <summary>
+    /// Provides extension methods to work with Worker Middleware against a <see cref="IHostBuilder"/>.
+    /// </summary>
+    public static class WorkerMiddlewareWorkerApplicationBuilderExtensions
+    {
+        /// <summary>
+        /// Configures the <see cref="IFunctionsWorkerApplicationBuilder"/> to use the default set of middleware used by the worker, in the following order:
+        /// <list type="number">
+        ///     <item>
+        ///         <description><see cref="OutputBindingsMiddleware"/></description>
+        ///     </item>
+        ///     <item>
+        ///         <description><see cref="FunctionExecutionMiddleware"/></description>
+        ///     </item>
+        /// </list>
+        /// </summary>
+        /// <param name="builder">The <see cref="IFunctionsWorkerApplicationBuilder"/> to configure.</param>
+        /// <returns>The configured <see cref="IFunctionsWorkerApplicationBuilder"/>.</returns>
+        public static IFunctionsWorkerApplicationBuilder UseDefaultWorkerMiddleware(this IFunctionsWorkerApplicationBuilder builder)
```

Comment:
new method to be explicit about the default. Previously, the call to the execution middleware was registering the output bindings one. Will update PR with the call to this instead.